Tasks/Maven/task.json

```diff
@@ -236,7 +236,7 @@
             "type": "connectedService:Generic",
             "label": "SonarQube Endpoint",
             "required": true,
-            "helpMarkDown": "The endpoint that specifies the SonarQube server to use",
+            "helpMarkDown": "The SonarQube server generic endpoint",
```

Comment:
I had a user asking what type of endpoint she should create. So I'm making this clearer.